lib/components/viewers/realtime-status-label.tsx

```diff
@@ -117,45 +120,49 @@ const RealtimeStatusLabel = ({
       withBackground={withBackground}
     >
       {renderedTime}
-      <MainContent>
-        <FormattedRealtimeStatusLabel
-          minutes={
-            isEarlyOrLate ? (
-              <DelayText>
-                <FormattedDuration
-                  duration={Math.abs(delay)}
-                  includeSeconds={false}
-                />
-              </DelayText>
-            ) : (
-              <>{null}</>
-            )
-          }
-          // @ts-ignore getTripStatus is not typed yet
-          status={STATUS[status].label}
-        />
-        {isEarlyOrLate && (
-          <InvisibleAdditionalDetails>
-            <FormattedMessage
-              id="components.MetroUI.originallyScheduledTime"
-              values={{
-                originalTime: (
-                  <FormattedTime timeStyle="short" value={originalTime} />
-                )
-              }}
-            />
-          </InvisibleAdditionalDetails>
-        )}
-      </MainContent>
+      {showScheduleDeviation && (
+        <MainContent>
+          <FormattedRealtimeStatusLabel
+            minutes={
+              isEarlyOrLate ? (
+                <DelayText>
+                  <FormattedDuration
+                    duration={Math.abs(delay)}
+                    includeSeconds={false}
+                  />
+                </DelayText>
+              ) : (
+                <>{null}</>
+              )
+            }
+            // @ts-ignore getTripStatus is not typed yet
+            status={STATUS[status].label}
+          />
+          {isEarlyOrLate && (
+            <InvisibleAdditionalDetails>
+              <FormattedMessage
+                id="components.MetroUI.originallyScheduledTime"
+                values={{
+                  originalTime: (
+                    <FormattedTime timeStyle="short" value={originalTime} />
+                  )
+                }}
+              />
+            </InvisibleAdditionalDetails>
+          )}
```

Comment:
This block still needs to be rendered in all cases. (It makes voice over say what the original time was, after enunciating the updated time first.)